basicsettings.ml

```diff
@@ -24,14 +24,35 @@ let concurrent_server = Settings.add_bool ("concurrent_server", true, `System)
 let printing_types = Settings.add_bool ("printing_types", true, `User)
 
 (** Name of the file containing the prelude code. *)
-let prelude_file = 
+let prelude_file =
   let prelude_dir = match Utility.getenv "LINKS_LIB" with
-      None -> Filename.dirname Sys.executable_name
+    (* If user defined LINKS_LIB then it takes the highest priority *)
     | Some path -> path
+    | None ->
+        (* If LINKS_LIB is not defined then we search in current directory *)
+        let current_dir = Filename.dirname Sys.executable_name in
```

Comment:
`current_dir` sounds like current working directory. This is not what is intended here. The idea is to get the path to the links executable (wherever it is) and find the prelude in the same folder.

(I just found this https://github.com/dbuenzli/logs/issues/14 which suggests that that would not work if links is bytecompiled. We might want to check/fix that.)